packages/suite-desktop-native/src/winHelloChildProcess.ts

```diff
@@ -0,0 +1,175 @@
+/**
+ * Worker thread module for Windows Hello native binary operations
+ * This runs in a separate worker thread to isolate the dlopen operations
+ */
+
+import fs from 'fs';
+import * as os from 'os';
+import path from 'path';
+// Child process for Windows Hello functionality
+
+import type { IPCMessage, IPCRequest, IPCResponse } from './types';
+import type * as WinHelloTypes from './win_hello.d';
+
+const errorMessageToStandardError = (errorMessage: string) => {
+    switch (errorMessage) {
+        case 'Canceled':
+            return 'Authentication canceled.';
+        default:
+            return errorMessage;
+    }
+};
+
+class WinHelloChildProcess {
+    private winHello: typeof WinHelloTypes | null = null;
+    private isWindows: boolean;
+
+    constructor() {
+        this.isWindows = os.platform() === 'win32';
+        const resourcesPath = process.env.RESOURCES_PATH;
+        if (!resourcesPath) {
+            console.error(
+                '[WinHelloChildProcess] resourcesPath not provided in environment variable',
+            );
+            throw new Error('resourcesPath not provided in environment variable');
+        }
+
+        this.initialize(resourcesPath);
+    }
+
+    private initialize(resourcesPath: string) {
+        if (!this.isWindows) {
+            return;
+        }
+
+        const binaryPath = path.join(resourcesPath, 'bin/win_hello.node');
```

Comment:
didn't notice it before, but I suggest this is safer on windows
```suggestion
        const binaryPath = path.join(resourcesPath, 'bin', 'win_hello.node');
```